src/main/mulmo/handler.ts

```
@@ -113,7 +113,7 @@
       }
     };
 
-    await generateBeatImage({ index, context, settings, forceImage, forceMovie, callbacks: [graphaiCallbacks] });
+    await generateBeatImage({ index, context, settings: settings.APIKEY ?? {}, forceImage, forceMovie, callbacks: [graphaiCallbacks] });
```

ESLint

[error] Replace `·index,·context,·settings:·settings.APIKEY·??·{},·forceImage,·forceMovie,·callbacks:·[graphaiCallbacks]` with `⏎······index,⏎······context,⏎······settings:·settings.APIKEY·??·{},⏎······forceImage,⏎······forceMovie,⏎······callbacks:·[graphaiCallbacks],⏎···`

(prettier/prettier)